app/keyboard.py

```diff
@@ -0,0 +1,89 @@
+"""
+This file will create button structures that we will use
+"""
+from telegram import InlineKeyboardButton, \
+    InlineKeyboardMarkup, ParseMode, KeyboardButton, ReplyKeyboardMarkup, ReplyKeyboardRemove
+
+"""
+Buttons from the bottom menu
+"""
+ADD_DEBT_KEYBOARD = "ADD DEBT"
+PAY_DEBT_KEYBOARD = "PAY DEBT"
+LIST_DEBT_KEYBOARD = "LIST OF DEBTS"
+INFO_KEYBOARD = "INFORMATION"
+
+"""
+Menu buttons that will be in the chat
+"""
+KEYBOARD_INLINE = [
```

Comment:
This is not a proper data type here, why can't you use e.g. dictionary?
The next code of using it is hard to read.

Reply:
Fixed it